### DISCUSSION TITLE
Change CrazyRadio Channel Without a CrazyFlie Connected

### DISCUSSION THREAD

**Discussion by TigeyJewellAlibhai** [bitcraze/discussions#531] [Q&A]
Hi all,

Is there a way to simply change the channel of the crazyradio? I have a bunch of different crazyflies that I put on different channels, and I want to be able to select one to fly, and then set my crazyradio to match its channel. Is this possible?

Thanks!

**Comment by tobbeanton** [comment]
I think you have to elaborate a bit more on this. How do you want to fly them?

**Reply by TigeyJewellAlibhai** [reply]
Thank you for the response! I have 9 crazyflies that are on channels 10, 20, 30, .. 90 and only a couple of crazyradios. My goal is that I can pick up  any single crazyflie, and set my crazyradio to the corresponding channel. Someone else can pick up a different crazyflie, and set their radio to the corresponding channel. Each radio is only ever flying one drone, but to avoid interference they are all on different channels. So my question is, if I pick up drone 4 (set to channel 40) but my crazyradio is on the default channel 80, how to I change the crazyradio channel to channel 40?

**Reply by TigeyJewellAlibhai** [reply]
I am using the xubuntu VM in virtualbox on a windows laptop, and an xbox controller to control

**Reply by krichardsson** [reply]
You set the channel when you connect to the Crazyflie. 

If you are using the python client you enter the **address** in the field at the top left, click `Scan` and a list of the Crazyflies using that address should appear in the drop down box. Pick the one with the **channel** you want to connect to and click `Connect`.

If you are using the python lib, use the appropriate URI when connecting. Check out the [examples](https://github.com/bitcraze/crazyflie-lib-python/tree/master/examples) , you will see code like:

``` python
uri = 'radio://0/80/2M/E7E7E7E7E7'
with SyncCrazyflie(uri) as scf:
    ## Do stuff
```

In this case 80 is the channel
